cli/src/text_util.rs

```diff
@@ -35,6 +37,33 @@ pub fn split_email(email: &str) -> (&str, Option<&str>) {
     }
 }
 
+pub fn render_copied_path(source: &str, target: &str) -> String {
+    if source == target {
+        return target.into();
+    }
```

Comment:
Nit: move this into the caller for now.

Reply:
I don't like that.  As it stands you can always pass strings to this and it will generate the right thing.  If you don't have this check, it will do something weird for the of equal strings.

Reply:
Basically the caller can handle the no-op or it should be documented somewhere, as it currently is a bit magic.

Reply:
added docs, there was already a test for this behavior.